firefox_cloud_services_ToS/cs.md

```diff
@@ -13,39 +13,39 @@ Tato úvodní část je shrnutím níže uvedených ustanovení. Toto shrnutí s
 
     Tyto podmínky poskytování služeb (dále jen „<u>Podmínky</u>”) upravují vaše používání Firefox cloud services, souboru online služeb poskytovaných společností Mozilla (dále jen „Služby”).
 
-2. #### Účty Firefox
+2. #### Účet Firefoxu
 
-    Pro používání některých Služeb budete muset založit účet Firefox.  Během registrace budete požádáni o nastavení hesla. Jste povinni uchovat své heslo v důvěrnosti a pro činnosti prováděné prostřednictvím vašeho účtu Firefox. Společnost Mozilla nenese odpovědnost za žádné ztráty, k nimž dojde během neoprávněného používání vašeho účtu Firefox.
+    Pro používání některých Služeb si budete muset založit účet Firefox. Během registrace budete požádáni o nastavení hesla. Jste povinni uchovat své heslo v důvěrnosti a nesete odpovědnost za činnosti prováděné prostřednictvím vašeho účtu Firefoxu. Společnost Mozilla nenese odpovědnost za žádné ztráty, k nimž dojde během neoprávněného používání vašeho účtu Firefoxu.
```

Comment:
"účet Firefox" => "účet Firefoxu".
Otherwise looks good

Reply:
Firefox is a brand name, it needs to remain in English, per branding guideline. I will let @MikkCZ comment on your suggestions.

Reply:
Hi @peiying2. "Firefox" is a brandname, but from what I know, "Firefox Accounts" are allowed to be localized. In [bug 1317086](https://bugzilla.mozilla.org/show_bug.cgi?id=1317086) we have agreed on translating "Firefox Accounts" as "účet Firefoxu" (the Firefox name is declined there) and all the material already refer to it the new name for some time. In Czech it's impossible to have a natural translation for Firefox Accounts without declension.

Reply:
ah, sorry for injecting comments not knowing the background.  Sounds good.

Reply:
No problem. :) I see this as an issue and I often need to check the documentation to remind, what we should translate and what we should not...

I will now go ahead an address the issues pointed above.